bundles/models/model.go

```diff
@@ -149,8 +149,9 @@ func (m *Model) GetUUID() *string {
 	return m.UUID
 }
 
-func (m *Model) toStorage(v uint64) storage.Resource {
-	return storage.NewResource(*m.GetUUID(), storage.KindModels, *m.GetOwner(), v)
+// newModelStorageResource creates a new storage.Resource for the given model associated to the given version.
+func newModelStorageResource(m *Model, v uint64) storage.Resource {
```

Comment:
I think `new` in the function name is implicit here, and explicitly mentioned in the documentation. Unless there's any naming collisions, we can drop it.

Reply:
I usually prefer keeping function names as operations: do, create, update. I'd prefer keeping the `new` to follow that rule, although I'm just aiming for consistency, we can change it if you want.

Reply:
How about `modelToStorageResource`? Or (even more verbose) `castModelToStorageResource`?

Reply:
What about `parse...`?

Reply:
`parse` makes me thing that you are converting from text into a language representation. I know we've used this in the past and I believe I raised the same point.

Reply:
I'm going with `cast`, and moved the helper func to the `common_resources` folder where it can be used with the `Resource` interface.